doc/concepts/index.rst

```diff
@@ -0,0 +1,35 @@
+.. _concepts:
+
+Concepts
+========
+
+Pachyderm enables you to create data pipelines that enable you
+continuously track changes that data scientists, developers, or other
+members of your team make to datasets. In other words, Pachyderm
+is a version-control platform for your data similar to what Git is
+for source code.
+
+Data scientints and researches often make changes to
+data in batches and phases. Often, they need to go back in time to
+revisit previous results or combine these data results with other
+data in different variants. Pachyderm helps solving the issues
```

Comment:
grammar: "Pachyderm helps solving"
typo: beginning
Grammar: "beginning of time"

Following the previous comment, You're saying that Pachyderm help solve issues with repro and data prov, but haven't actually motivated those problems. 

"Data is changing" does not directly imply "I have reproducibility and provenance issues." That's a big leap. As with above, I think Nick has standard "what is pachyderm" wording that will cover this already and both of these 1st two paragraphs can be replaced with that.